app/components/questionnaire/file-field.js

```diff
@@ -27,15 +28,16 @@ const FileField = DDSProjectField.extend({
       }
     })),
   formatSettings: null,  // settings based on cwl type and format
-  displayFieldName: Ember.computed('fieldName', function() {
-    const fieldName = this.get('fieldName');
-    if(fieldName) {
-      return fieldName.capitalize();
+  fieldName: null,
+  fieldLabel: null,
+  displayLabel: Ember.computed('fieldLabel', 'fieldName', function () {
```

Comment:
Can we refactor this to a common base class or Mixin rather than maintaining the same logic in two places?